MavenArtifacts.md

```diff
@@ -0,0 +1,83 @@
+# WPILib Maven Artifacts
+
+WPILib publishes its built artifacts to our maven server for use by downstream projects. This document explains these locations, and the meanings of artifact names, classifiers and versions.
+
+## Repositories
+We provide 2 repositories. These repositories are:
+
+* (Release)     http://first.wpi.edu/FRC/roborio/maven/release/
+* (Development) http://first.wpi.edu/FRC/roborio/maven/development/
+
+The release repository is where official WPILib releases are pushed.
+The development repository is where development releases of every commit to `master` is pushed.
```

Comment:
Instead of making master a code quote, can we link here: https://github.com/wpilibsuite/allwpilib/tree/master